src/Runtime/Secrets.php

```diff
@@ -76,4 +74,27 @@ protected static function parseSecrets(array $secrets)
             return [$segments[count($segments) - 1] => $secret['Value']];
         })->all();
     }
+
+    protected static function setEnvironmentVariables(array $variables)
+    {
+        foreach ($variables as $key => $value) {
+            if (Str::startsWith($key, 'DOT_ENV_')) {
```

Comment:
Basically, if the secret names starts `DOT_ENV_`, we parse it as a dot env file :)